otterdog/adoptium.jsonnet

```diff
@@ -725,6 +724,17 @@ orgs.newOrg('adoptium') {
         },
       ],
     },
+    orgs.newRepo('temurin-vdr-generator') {
+      allow_auto_merge: true,
+      default_branch: "main",
+      description: "Scripts for generating Vulnerability Disclosure Reports",
+      topics+: [
+        "secure-dev"
+      ],
+      branch_protection_rules: [
```

Comment:
also not sure if this is set by default CC @netomi

Reply:
there is no default branch protection rule setup by default.

For the vert.x project we did something like that: 

https://github.com/eclipse-vertx/.eclipsefdn/blob/main/otterdog/eclipse-vertx.jsonnet#L18